lib/luaxtable.c

```diff
@@ -31,6 +31,11 @@
 #include <lua.h>
 #include <lauxlib.h>
 #include <lunatik.h>
+#include "luadata.h"
+#include "luarcu.h"
+#include "luaxtable.h"
+
+#define XTABLE_DEFAULT_SIZE 1024
```

Comment:
please, use `LUAXTABLE` prefix for this module definitions to not get mixed up with the actual `XTABLE` ones.. also, if we are going to use LUARCU default value, let's just use its define instead of creating another one..